woocommerce/google-listings-and-ads
# Remove delete operations for campaign budget and ads group

### Changes proposed in this Pull Request:



Closes #1362. This PR remove delete operations of campaign budget and ads group. For detailed information please see #1362.

### Detailed test instructions:


1. Create a campaign.
2. Remove the newly created campaign.
3. From the GLA dashboard, this campaign should not appear.
4. From the Google Ads dashboard, this campaign's status should be `Removed` and the budget amount is still visible.
    <img width="403" alt="Screenshot 2022-03-25 at 18 14 10" src="https://user-images.githubusercontent.com/914406/160101527-5fdf5539-0beb-4a23-acfe-8847f16168c8.png">
5. From the Google Ads dashboard, this campaign's `ad group` should still be visible and its status should be `Not eligible`.
    <img width="371" alt="Screenshot 2022-03-25 at 18 13 44" src="https://user-images.githubusercontent.com/914406/160101462-92f22836-76a7-4c6d-b7b4-a6d2692ebf75.png">
6. From the Google Ads dashboard, this campaign's `ad` should still be visible and its status should be `Not eligible`.
    <img width="787" alt="Screenshot 2022-03-25 at 18 14 34" src="https://user-images.githubusercontent.com/914406/160101591-e815bcc6-d8b1-4796-9cd2-b48246557ee4.png">
7. From the Google Ads dashboard, this campaign's `location` is still visible.
    <img width="251" alt="Screenshot 2022-03-25 at 18 18 42" src="https://user-images.githubusercontent.com/914406/160102274-77d75b4e-b146-48f2-95fb-31b2ae1caa9b.png">


### Changelog entry

> Update - Remove delete operations for campaign budget and ads group.
